docs/manual/docs/cmd/spo/list/list-webhook-remove.md

```diff
@@ -0,0 +1,49 @@
+# spo list webhook remove
+
+Removes the specified webhook from the listt
+
+## Usage
+
+```sh
+spo list webhook remove [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`|output usage information
+`-u, --webUrl <webUrl>`|URL of the site where the list to remove the webhook from is located
+`-l, --listId [listId]`|ID of the list from which the webhook should be removed. Specify either `id` or `title` but not both
+`-t, --listTitle [listTitle]`|Title of the list from which the webhook should be removed. Specify either `id` or `title` but not both
+`--confirm`|Don't prompt for confirming removing the webhook
+`-o, --output [output]`|Output type. `json|text`. Default `text`
+`--verbose`|Runs command with verbose logging
+`--debug`|Runs command with debug logging
+
+!!! important
+    Before using this command, log in to SharePoint, using the [spo login](../login.md) command.
+
+## Remarks
+
+To remove a webhook from a list, you have to first log in to SharePoint using the [spo login](../login.md) command, eg. `spo login https://contoso.sharepoint.com`.
+
```

Comment:
Missing the remark about the error you get when the ID doesn't point to an existing webhook